hack/make-rules/verify.sh

```diff
@@ -73,6 +73,17 @@ function is-quick {
   return 1
 }
 
+function is-explicitly-chosen {
+  local name="${1#verify-}"
+  name="${name%.*}"
+  for e in ${VERIFY_CHECKS}; do
```

Comment:
maybe `VERIFY_CHECKS` should be documented somewhere, e.g. in the `make verify` help text?

Reply:
yeah, I think it should be [documented](https://github.com/kubernetes/kubernetes/blob/209cdd90489a5af11f2f124e592c57df82a3da28/build/root/Makefile#L110-L127). I'm also wondering if we should follow the `WHAT=` pattern that a lot of other makerules follow, and have WHAT map in this case to the VERIFY_CHECKS var, like BRANCH already does to KUBE_VERIFY_GIT_BRANCH

Reply:
WHAT= SGTM

Reply:
Done.